utils/parsing.py

```diff
@@ -3,10 +3,13 @@
 
 
 def ParseModifiers(modifier_st: str):
+    if modifier_st == '':
+        return modifier_st
     return f"{'+' if modifier_st[0] not in ['+', '-'] else ''}{modifier_st}"
```

Comment:
```suggestion
    return f"{'+' if modifier_st and modifier_st[0] not in ['+', '-'] else ''}{modifier_st}"
```
This will make sure it returns blank for the first {} and so has the same result.